mautic/mautic
# TPROD-333 - MauticClearbitBundle - Convert PHP Templates to Twig Templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. 



* Navigate to Settings > Plugins and click on the Clearbit plugin. Verify the modal renders correctly
* (With plugin enabled) Navigate to the contacts page (`/s/contacts`) and using the dropdown for a specific contact, select **Lookup using Clearbit** and verify the modal renders correctly
* (With the plugin enabled) Navigate to the contacts page (`/s/contacts`) and select 2 or more contacts. Using the batch dropdown, select **Lookup using Clearbit** and verify the modal renders correctly

**NOTE**: Because this does not change anything related to the actual integration, there is no need to verify it can connect and process data via Clearbit. This also means that you can enable the plugin using random text as the API Key.
